test/unittest/xmlparser/CMakeLists.txt

```diff
@@ -135,6 +135,7 @@ target_compile_definitions(XMLProfileParserTests PRIVATE FASTRTPS_NO_LIB
     $<$<BOOL:${INTERNAL_DEBUG}>:__INTERNALDEBUG> # Internal debug activated.
     )
 target_include_directories(XMLProfileParserTests PRIVATE
+    ${PROJECT_SOURCE_DIR}/src/cpp
```

Comment:
Why is this change included if the XML Parser tests have not been modified?

Reply:
At some point, it was needed for including dependencies on `<rtps/....>` but since we moved SenderResource.h to `fastdds/...>` it is no longer needed